build_tools/packaging/python/templates/rocm/src/rocm_sdk/tests/libraries_test.py

```diff
@@ -55,10 +55,21 @@ def testSharedLibrariesLoad(self):
 
         for so_path in so_paths:
             with self.subTest(msg="Check shared library loads", so_path=so_path):
+
+                # For Windows compatibility, we first preload libraries (DLLs)
+                # that are not co-located. Specifically this is for
+                # the "libraries" like hipfft, rocblas, etc. which are siblings
+                # in '_rocm_sdk_libraries_gfx####/bin' while the "compiler" is
+                # in '_rocm_sdk_core/bin'
+                preload_command = "import rocm_sdk; rocm_sdk.preload_libraries('amd_comgr', 'amdhip64', 'hiprtc');"
```

Comment:
Can you add an issue to add deps to the library definitions so that we can code the preloader to recursively get all dependent libraries vs needing to hardcode it like this at the callsite?

Reply:
Filed https://github.com/ROCm/TheRock/issues/996 and will link to it in the code.